### ISSUE TITLE
Unrecognizable text in app, when add comment and ticket text from glpi web interface

### ISSUE THREAD

**Issue by trisosamu** [clesssalvein/GlpiClient#3]
![image](https://user-images.githubusercontent.com/78987040/213714158-66ad6503-a6ba-4a34-9e0c-dc474e11f79d.png)
![image](https://user-images.githubusercontent.com/78987040/213713900-79fd3189-493b-45c3-8b26-9ce409c43d4b.png)

Don't hate me ahah

**Comment by trisosamu** [comment]
Instead, preview works correctly

![image](https://user-images.githubusercontent.com/78987040/213714512-0d6a1ce3-b7ed-4174-834a-447b632bf785.png)

**Comment by clesssalvein** [comment]
Yes, unfortunately we allow our users to work with the GLPI-server only through this app, so we haven't such problems. The problem is in the html markup. I think that I can solve the problem, but I need more time for it and I can't tell when exactly I can do it.

**Comment by clesssalvein** [comment]
In release [GlpiClient v0.304](https://github.com/clesssalvein/GlpiClient/releases/tag/v0.304) I made some fixes, but not sure that I fixed everything.

**Comment by trisosamu** [comment]
Works with the latest update. 
Thank you

Only things are the spaces. (from email). from web interface it's perfect
![image](https://user-images.githubusercontent.com/78987040/213984162-d71b2ca0-be5b-4f95-b64a-6e04d7dab8de.png)
